pkg/kubelet/kubelet.go

```diff
@@ -705,31 +704,6 @@ func containerRef(pod *api.Pod, container *api.Container) (*api.ObjectReference,
 	return ref, nil
 }
 
-// setRef stores a reference to a pod's container, associating it with the given docker id.
```

Comment:
SGTM, yeah we'll re-examine as we go. My motivation was to have the code in the same file. It doesn't have to be exported. Doesn't hurt for now.